test/optim/test_optim.py

```diff
@@ -845,30 +845,33 @@ def test_multi_tensor_optimizers_with_varying_tensors(self):
         ]
         self._test_derived_optimizers_varying_tensors(optimizer_pairs_with_flags, "foreach")
 
-    def test_fused_optimizers(self):
-        optimizer_pairs_with_flags = tuple(itertools.product(
+    @property
+    def _fused_optimizer_configs(self):
+        return list(itertools.product(
             (optim.Adam, optim.AdamW),
             (
                 dict(weight_decay=1., amsgrad=False),
                 dict(weight_decay=1., amsgrad=True),
                 dict(weight_decay=0., amsgrad=False),
                 dict(weight_decay=0., amsgrad=True),
             ),
+        )) + list(itertools.product(
+            (optim.SGD,),
+            [
+                {"lr": 0.1, "momentum": 0.0, "dampening": d, "weight_decay": w, "nesterov": n}
+                for d, w, n in itertools.product((0.0, 0.5), (0.0, 0.5), (False,))
```

Comment:
Minor nit but it seems like this entry and the next entry in the list could be combined by adding another tuple (0.0, 0.5) for momentum.